src/analyserwaveform.cpp

```diff
@@ -16,6 +16,7 @@ AnalyserWaveform::AnalyserWaveform(ConfigObject<ConfigValue>* pConfig) :
         m_skipProcessing(false),
         m_waveformData(NULL),
         m_waveformSummaryData(NULL),
+        m_stride(),
```

Comment:
I'd prefer that too actually. At
```
m_stride({0,0.0,0.0,0,0,{},{},{},{},0.0}),
```
the compiler yells
```
src/analyserwaveform.cpp:21:33: warning: extended initializer lists only available with -std=c++11 or -std=gnu++11 [enabled by default]
```
at me. (This is with GCC 4.8.2.)